### PR TITLE
Plenty of grammar+syntax+capitalizing on French language (fr.po)

### PR DESCRIPTION
**Pull request by GLLM**
Grammar & syntax corrections
Light rewording
Rationalize syntax : e.g.e  right-click & right click
Capilized : IL, EV, JP*G, ISO, DCP, ICC, EXIF, XMP
Did not capitalize "raw" because its not an abbreviation but a word.

### CODE REVIEW

**Comment by timur-davletshin** [comment]
EXIF should be spelled as Exif according to wikipedia.

File: po/fr.po [hunk 1]
```diff
@@ -3657,17 +3657,17 @@ msgstr "fichier de stockage"
 msgid "general"
 msgstr "général"
 
-#. ignoring of jpegs. hack while we don't handle raw+jpeg in the same directories.
+#. ignoring of JPEGs. hack while we don't handle raw+JPEG in the same directories.
```
Comment (TurboGit) [inline po/fr.po]: Maybe RAW+JPEG then, no?

File: po/fr.po [hunk 2]
```diff
@@ -8546,8 +8546,8 @@ msgstr ""
 "ne jamais toucher sauf si vous savez ce que vous faites."
 
 #: ../src/iop/filmicrgb.c:192
-msgid "filmic rgb"
-msgstr "filmique rvb"
+msgid "filmic RGB"
```
Comment (TurboGit) [inline po/fr.po]: You can't change this string as it is the one in the code. This is the "key" that dt will use at runtime to find the translation. Changing that breaks the translation.

File: po/fr.po [hunk 3]
```diff
@@ -6386,7 +6386,7 @@ msgstr "position verticale"
 
 #: ../src/iop/borders.c:1043
 msgid ""
-"select the vertical position ratio relative to left or right click and type "
+"select the vertical position ratio relative to left or right-click and type "
```
Comment (TurboGit) [inline po/fr.po]: You cannot change the main English string here. This should be done in the code and cannot be done at this stage as we are in string freeze.

File: po/fr.po [hunk 4]
```diff
@@ -9465,7 +9465,7 @@ msgstr "outil point: éditer, ajouter et supprimer des points"
 
 #: ../src/iop/liquify.c:3607
 msgid ""
-"ctrl-click: add node - right click: remove path\n"
+"ctrl-click: add node - right-click: remove path\n"
```
Comment (TurboGit) [inline po/fr.po]: no, see above.

File: po/fr.po [hunk 5]
```diff
@@ -9474,7 +9474,7 @@ msgstr ""
 #: ../src/iop/liquify.c:3609
 msgid ""
 "click and drag to move - click: show/hide feathering controls\n"
-"ctrl-click: autosmooth, cusp, smooth, symmetrical - right click to remove"
+"ctrl-click: autosmooth, cusp, smooth, symmetrical - right-click to remove"
```
Comment (TurboGit) [inline po/fr.po]: no, see above.

**Comment by GLLM** [comment]
> You need to revert all changes done in main English strings.

Oh. I'm very sorry about this, I'll remove my PR.
Sadly, I checked the de.po file and noticed that the english strings were capitalized differently, so I thought it was fine...

For the raw+JPG ... in my first commit comment, i thought about it but did not do it as JPEG are initials whereas RAW is not, just a word (afaik).

I'm sorry @TurboGit if my PR triggered some unnecessary work on your side with the english strings & the few miss I later saw you correct.

I will remove my PR, so that I don't fill the commits list with more crap whereas there are plenty of useful code addition pending.

Merci
GLLM

**Pull request closed without merging**